assets/locales/service.cy.toml

```diff
@@ -21,21 +21,26 @@ other = "Mireinio'r canlyniadau"
 description = "Content types to show"
 other = "Mathau o gynnwys i'w dangos"
 
+[ContentType]
+description = "Content type"
+one = "math o gynnwys"
```

Comment:
I suspect the first letter of a number of these entries should be capitalised in Welsh as they are in English

Reply:
yes indeed! I have corrected.